hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/RouterMetrics.java

```diff
@@ -342,6 +378,34 @@ private RouterMetrics() {
     listReservationLatency =
         registry.newQuantiles("listReservationLatency",
             "latency of list reservation timeouts", "ops", "latency", 10);
+
+    getAppActivitiesLatency =
+        registry.newQuantiles("getAppActivitiesLatency",
+            "latency of get app activities timeouts", "ops", "latency", 10);
+
+    getAppStatisticsLatency =
+        registry.newQuantiles("getAppStatisticsLatency",
+            "latency of get app statistics timeouts", "ops", "latency", 10);
+
+    getAppPriorityLatency =
+        registry.newQuantiles("getAppPriorityLatency",
+            "latency of get app priority timeouts", "ops", "latency", 10);
+
+    getAppQueueLatency =
+        registry.newQuantiles("getAppQueueLatency",
+            "latency of get app queue timeouts", "ops", "latency", 10);
+
+    getUpdateQueueLatency =
+        registry.newQuantiles("getUpdateQueueLatency",
+            "latency of update app queue timeouts", "ops", "latency", 10);
+
+    getAppTimeoutLatency =
+        registry.newQuantiles("getAppTimeoutLatency",
+            "latency of get apptimeout timeouts", "ops", "latency", 10);
+
+    getAppTimeoutsLatency =
```

Comment:
You can tweak the spacing and compress this a little:
```
    getAppTimeoutsLatency = registry.newQuantiles("getAppTimeoutsLatency",
        "latency of get apptimeouts timeouts", "ops", "latency", 10);
```

No need to do it for the old ones.

Reply:
Thanks for your suggestion, I will fix it.